docs/content/how-to/azure/create-azure-cluster_on_aks.md

```diff
@@ -0,0 +1,163 @@
+# Create an Azure Hosted Cluster on AKS
+This document describes how to set up an Azure Hosted Cluster on an AKS management cluster with an ExternalDNS setup.
+
+If you already have an existing AKS cluster up and running, you can jump to the [Setup ExternalDNS](#setup-externaldns)
+
+## Prerequisites
+To create an AKS cluster, first set up your constants
+```
+RG="aks-test"
+LOCATION="eastus"
+AKS_CLUSTER_NAME="test"
+```
+
+Log in to your Azure account through the CLI
+`az login`
```

Comment:
Can we add az cli as a prereq and maybe a link where to get it from ?

Reply:
Sure